lib/vmdb/plugins.rb

```diff
@@ -40,6 +40,17 @@ def ansible_content
       end
     end
 
+    def ansible_runner_content
+      @ansible_runner_content ||= begin
+        map do |engine|
+          content_dir = engine.root.join("content", "ansible_tmp")
+          next unless File.exist?(content_dir.join("requirements.yml"))
```

Comment:
Can we reuse the `content_directories` method?

Reply:
`content_directories` would do `content/ansible_tmp/*` which would return every file and dir in that directory.  For the purpose of seeding we only care about the role_file.  We could change `content_directories` to take the last component as an arg and default to `*` but we could override with .e.g. `requirements.yml`